databricks/koalas/tests/test_dataframe.py

```diff
@@ -88,6 +88,34 @@ def test_dataframe(self):
         index_cols = pdf.columns[column_mask]
         self.assert_eq(kdf[index_cols], pdf[index_cols])
 
+    def test_insert(self):
+        pdf = pd.DataFrame([1, 2, 3])
+        kdf = ks.from_pandas(pdf)
+
+        # Inserts a Scalar
+        kdf.insert(1, "b", 10)
+        pdf.insert(1, "b", 10)
+        self.assert_eq(kdf.sort_index(), pdf.sort_index(), almost=True)
```

Comment:
Why `almost=True`?

Reply:
The inserted column dtype is different between Koalas and pandas.

After removing `almost=True`:
```
Attribute "dtype" are different
[left]:  int32
[right]: int64

Left:
   0   b
0  1  10
1  2  10
2  3  10
0    int64
b    int32
dtype: object

Right:
   0   b
0  1  10
1  2  10
2  3  10
0    int64
b    int64
dtype: object
```